c/dev-tools/dev-cli.c

```diff
@@ -124,7 +124,11 @@ print_ld_matrix(tsk_treeseq_t *ts)
                 fatal_library_error(ret, "get_site");
             }
             printf("%d\t%f\t%d\t%f\t%.3f\n",
-                (int) sA.id, sA.position, (int) sB.id, sB.position, r2[k]);
+                (int) sA.id,
```

Comment:
It's not obvious what the rules are for going into one-arg-per-line mode. Although in this case it's probably a good thing all right.